pkg/rke/k8s_rke_system_images.go

```diff
@@ -9848,6 +9890,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.8.2
```

Comment:
Timing is not 100% clear but most likely this will be released before v2.8.2 so let's update it to v2.8.1 here.
```suggestion
		// Enabled out of band post v2.8.1
```

Reply:
updated